index.html

```diff
@@ -0,0 +1,174 @@
+
+                       <div class="sign">
+                         <h3>2022.10.17(MON) ~ (FRI)</h3>
+                         <h5>@information share Karangwa</h5>
+                       </div>
+                      </div>
+             </header>
+             <section>
+                <aside class="dishes">
+                  <div class="title">
+                   <h4>SIGNATURE DISHES</h4>
+                   <div class="bar"></div>
+                  </div>
+                   
+                   <div class="cards-container">
+                     <div class="cards">
+                         <img src="./components/service1.jpg" alt="service1 image">
+                         <div>
+                              <h4>Grill</h4>
+                              <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                              </p>
+                         </div>
+                     </div>
+                     <div class="cards">
+                         <img src="./components/service2.jpg"  alt="service2 image">
+                         <div>
+                              <h4>Meat</h4>
+                              <p>
+                                  Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                               </p>
+                         </div>
+                     </div>
+                     <div class="cards">
+                          <img src="./components/service3.jpg" alt="service3 image">
+
+                         <div>
+                              <h4>Hamburger</h4>
+                              <p>
+                                 Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                              </p>
+                          </div>
+                     </div> 
+                     <div class="cards">
+                         <img src="./components/bg2.jpg" alt="service4 image">
+                          <div>
+                              <h4>Salade</h4>
+                              <p>
+                                 Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                              </p>
+                          </div>
+                     </div> 
+                     <div class="cards">
+                       <img src="./components/grill.jpg" alt="service4 image">
+                        <div>
+                            <h4>Grill</h4>
+                            <p>
+                               Lorem ipsum dolor sit amet consectetur, adipisicing elit.  
+                            </p>
+                        </div>
+                   </div> 
+                   </div>
+                   <div class="dish-link">
+                     <a href="#">SEE THE WHOLE SIGNATURE</a>
+                   </div>
+                   <div class="dish-link2">
+                     <a href="#">Join CC Global Summit 2015</a>
+                   </div>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the main section in the mobile view is semantic, all the columns should take the same space

![Screenshot_20221022_170009](https://user-images.githubusercontent.com/25227022/197349214-bb48dbd1-0f4f-470a-a6e5-a3e2d20e403c.png)